app/controllers/ProductController.scala

```diff
@@ -0,0 +1,34 @@
+package controllers
+
+import models.Product
+import play.api.Environment
+import play.api.libs.json.Json
+import play.api.mvc.{AbstractController, ControllerComponents}
+
+import java.time.LocalDateTime
+import javax.inject.Inject
+
+// Controller class for whenever a user enters the URL for the bakery.
+class ProductController @Inject() (
+                                    cc: ControllerComponents,
+                                    environment: Environment
+                                  ) extends AbstractController(cc) {
+  // Create the format for the product in JSON form.
+  implicit val productFormat = Json.format[Product]
+
+  // When the user route to <root>/pass-bakery/status then this will return a status JSON
+  def getAll = Action {
+
+    // Get the current server time when the user enters the link
+    val curDateTime = LocalDateTime.now()
+
+    // Get the current environment
```

Comment:
This comments tell me what you are doing. It is more helpful to get used to writing comments specifying `why` you are doing something. As it is, remove these single line comments, they are not helpful